test/test_playbooks/content_view.yml

```diff
@@ -1,19 +1,46 @@
 ---
-- hosts: localhost
-  tasks:
-  - name: Load server config
-    include_vars:
-      file: server_vars.yml
-  - name: "Create/update katello content view"
-    katello_content_view:
+- hosts: tests
+  vars:
+    auth_params: &auth_params
```

Comment:
Sorry for the confusion here. This YAML referencing was an experiment of mine. It would have been really cool if it worked across file boundaries, but it does not.

Reply:
np